HIRS_ProvisionerTPM2/CMakeLists.txt

```diff
@@ -257,7 +257,7 @@ if (${DISTRIBUTION} STREQUAL "Ubuntu")
     set(CPACK_GENERATOR "DEB")
     set(CPACK_DEBIAN_PACKAGE_NAME "HIRSProvisionerTPM2.0")
     set(CPACK_DEBIAN_PACKAGE_SECTION "admin")
-    set(CPACK_DEBIAN_PACKAGE_DEPENDS "liblog4cplus-1.1-9(>=1.1.2), libcurlpp0(>=0.7), lshw")
```

Comment:
Should paccor be added here?

Reply:
Added it.

Reply:
Let's hold up for a second here. We should probably come to some sort of consensus regarding including paccor as a depedency in the RPMs and DEBs. If the user would have to ignore dependencies to install the package just for the sake of Paccor, then it shouldn't be included. More likely we should write a separate issue to detect Paccor (similar to the issue to detect TPM2-Abrmd).

Reply:
Pending the answer to my last question, this should be fine.